incubator/binding-pgsql/src/main/java/io/aklivity/zilla/runtime/binding/pgsql/parser/listener/SqlAlterStreamTopicListener.java

```diff
@@ -29,6 +29,7 @@ public class SqlAlterStreamTopicListener extends PostgreSqlParserBaseListener
 {
     private final TokenStream tokens;
 
+    private String schema;
     private String name;
     private final List<AlterExpression> alterExpressions = new ArrayList<>();
```

Comment:
Please move `final` field up with other `final` fields and initialize in constructor as it is not a (pseudo-) constant.